hooks/environment

```diff
@@ -12,6 +12,22 @@ check_command() {
   fi
 }
 
+# the buildkite agent redactor has been added from version 3.66.0 and on. So we need to check it.
+# https://github.com/buildkite/agent/releases/tag/v3.66.0
+check_buildkite_version() {
```

Comment:
```suggestion
check_buildkite_version_for_redaction() {
```

?

Reply:
In fact I will add also the `_agent` since you propose a change, because version refers to agent.

Reply:
changed in 2cab9997094d5f493b5fd50e2ac765f66a5e37de